backend/api/src/test/kotlin/software/shonk/adapters/incoming/ShorkInterpreterControllerV1IT.kt

```diff
@@ -207,4 +207,68 @@ class ShorkInterpreterControllerV1IT() : AbstractControllerTest() {
         assertEquals(HttpStatusCode.Created, result3.status)
         assertEquals("3", result3.bodyAsText())
     }
+
+    @kotlinx.serialization.Serializable
+    data class LobbyInformation(
+        val lobbyId: Long,
+        val playersJoined: List<String>,
+        val gameState: String,
+    )
+
+    private suspend fun parseAllLobbiesResponse(
+        httpResponse: HttpResponse
+    ): MutableList<LobbyInformation> {
+        val body = httpResponse.bodyAsText()
+        val jsonObject = Json.parseToJsonElement(body).jsonObject
+
+        assert(jsonObject.containsKey("lobbies"))
+        return Json.decodeFromJsonElement<MutableList<LobbyInformation>>(jsonObject["lobbies"]!!)
+    }
+
+    suspend fun createLobby(client: HttpClient): Long {
+        val result = client.post("/api/v1/lobby")
+        assertEquals(HttpStatusCode.Created, result.status)
+
+        return result.bodyAsText().toLong()
+    }
+
+    @Test
+    fun `get all lobbies returns only default lobby if none have been created`() = runTest {
+        val result = client.get("/api/v1/lobby")
+
+        assertEquals(HttpStatusCode.OK, result.status)
+
+        val lobbies = parseAllLobbiesResponse(result)
+
+        assertEquals(1, lobbies.size)
+        assertEquals(LobbyInformation(0, emptyList(), "NOT_STARTED"), lobbies.first())
+    }
```

Comment:
Depending on how we decide to continue developing this is to be changed, but probably part of another branch / PR, so fine I guess